buildingmotif/dataclasses/model.py

```diff
@@ -296,3 +305,32 @@ def get_manifest(self) -> ShapeCollection:
         :rtype: ShapeCollection
         """
         return ShapeCollection.load(self._manifest_id)
+
+    def update_manifest(self, manifest: Union[ShapeCollection, "Library"]):
```

Comment:
Should update manifest only be a pointer copy? It seems like maybe a deep copy would make more sense? that way if you have a root library which you use as the starting point for your manifests it doesn't get the specific changes made for each model.

Reply:
Great point! I'm changing it to merge in the provided manifest's contents into the model's manifest graph. That way you get a copy of everything